WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/ProductCreationAIStartingInfoView.swift

```diff
@@ -275,13 +276,59 @@ private extension ProductCreationAIStartingInfoView {
     }
 }
 
+private extension ProductCreationAIStartingInfoView {
+    struct ToneOfVoiceView: View {
```

Comment:
❓ Can this be on a separate file for clarity?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/13225/commits/6c59cc29dedb9e30b0e33b4f4b78a9473b44a77d